lib/config/executor/container_vm.go

```diff
@@ -64,7 +64,7 @@ type ExecutorConfigCommon struct {
 	ID string `vic:"0.1" scope:"read-only" key:"id"`
 
 	// Convenience field to record a human readable name
-	Name string `vic:"0.1" scope:"hidden" key:"name"`
+	Name string `vic:"0.1" scope:"read-only" key:"name"`
```

Comment:
I put it in initially since I wanted to use the VCH name in the syslog tag. But I took away that change, and its only in there because now we actually set the VCH hostname properly because of this. I will run the rename tests to confirm if it breaks rename or not. If it does, I'll pull it out.